pkg/test/echo/server/forwarder/config.go

```diff
@@ -55,13 +55,24 @@ type Config struct {
 	urlPath                 string
 	method                  string
 	secure                  bool
+
+	// completed whether the Config is complete or still in progress
+	completed bool
```

Comment:
why did you need this?

Reply:
tbh I forget.. some issue about a race condition calling the setup twice. Does seem suspicious, let me look into it